themes/default/content/blog/resource-search/index.md

```diff
@@ -0,0 +1,110 @@
+5. **Data Export**: Export resource data to ingest in your data warehouse. This will be covered in more detail in a future blog post, stay tuned!
+
+## Query Syntax
+
+It can be helpful when picking up a new feature to see some example queries that you can edit for your organization. See below a list of example query syntax queries:
+
+See all resources that have not been modified in a year
+
+```modified:<=2022-05-18```
+
+See all resources with a certain tag in your cloud provider
+
+```type:aws:s3/bucket:Bucket tags.costcenter:1234```
+
+Find the Snowflake resources used in your data engineering team
+
+```package:snowflake stack:production team:data```
+
+Find resources in a project that aren't in a certain stack
+
+```package:aws project:pulumi-cloud-import-aws stack:-staging-us-west-2```
+
+## AI Assist Examples
+
+show me all virtual machines across all cloud providers
+
+```(type:aws:ec2/instance:Instance OR type:azure:compute:VirtualMachine OR type:gcp:compute:Instance)```
+
+show me all s3 buckets not tagged in production
+
+```(type:aws:s3/bucket:Bucket -tags.enviornment:production```
+
+show azure and azure native security groups
+
+```((type:azure:network:NetworkSecurityGroup OR type:azure-native:network:NetworkSecurityGroup)```
+
+## Advanced Filtering & Pulumi Teams support
+
+**The new advanced filtering menu!**
+![Advanced filtering screenshot](data.png)
+
+When navigating to the Resources page you will find your top 3 resource types, projects, stacks, teams and packages by resource count. We have extended this to show the top 10 when you click 'Show more'. This came as a direct byproduct of the feedback we have gotten from our customers that this feature added value upon first page load, and having more items would make it even more valuable.
+
+Customers who use [Pulumi Teams](/docs/pulumi-cloud/access-management/teams) for role based access control on Pulumi Cloud can now filter resources by team access. We have added a new advanced filter option for Teams, allowing you to see at a glance the teams in your organization with the most resources. You can query which teams have access to what resources, track resource usage by team by exporting your resource data and see in the search results table which teams have access to that resource.
+
+## Wrapping it up
+
+Resource Search is available to all Pulumi users for free, making it even easier to manage your cloud resources and gain valuable insights into your infrastructure. It's just one of the many ways Pulumi continues to innovate and support organizations in their journey to the cloud. Try Resource Search today and see the difference it can make in helping you explore and find resources in your organization.
```

Comment:
```suggestion
Resource Search is available to all Pulumi users for free, making it even easier to manage your cloud resources and gain valuable insights into your infrastructure. It's just one of the many ways Pulumi continues to innovate and support organizations in their journey to the cloud. Try Resource Search today and see the difference it can make in helping you explore and find resources in your organization.

[Login to Pulumi Cloud](https://app.pulumi.com)
```